### PR TITLE
PIN-113: another bnf bugs

### PR DESCRIPTION
**Pull request by Troi**

### CODE REVIEW

**Comment by Troi** [comment]
added replace /projects endpoint serialization

File: src/ProjectBundle/Mapper/ProjectMapper.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace ProjectBundle\Mapper;
+
+use BeneficiaryBundle\Mapper\AssistanceMapper;
+use BeneficiaryBundle\Repository\BeneficiaryRepository;
+use DistributionBundle\Entity\DistributionBeneficiary;
```
Comment (iarro) [inline src/ProjectBundle/Mapper/ProjectMapper.php:7]: Please pay attention to code style - unused imports

File: src/ProjectBundle/Mapper/ProjectMapper.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        $this->assistanceMapper = $assistanceMapper;
+        $this->beneficiaryRepo = $beneficiaryRepo;
+    }
+
+    public function toFullArray(?Project $project): ?array
+    {
+        if (!$project) {
+            return null;
+        }
+        $bnfCount = $this->beneficiaryRepo->countAllInProject($project);
```
Comment (iarro) [inline src/ProjectBundle/Mapper/ProjectMapper.php:50]: Please pay attention to code style - invalid codestyle

File: src/ProjectBundle/Mapper/SectorMapper.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace ProjectBundle\Mapper;
+
+use ProjectBundle\Entity\Donor;
+use ProjectBundle\Entity\Project;
```
Comment (iarro) [inline src/ProjectBundle/Mapper/SectorMapper.php:6]: unused imports

**Review by iarro — APPROVED** [review]

**Pull request merged**